contracts/PartyBid.sol

```diff
@@ -158,7 +158,7 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
         name = _name;
         symbol = _symbol;
         // validate token exists (ownerOf should revert if token doesn't exist)
-        IERC721Metadata(_nftContract).ownerOf(_tokenId);
```

Comment:
other places where we call into `IERC721Metadata(_nftContract)`: 
- `approve` inside `_fractionalizeNFT`, which is called if the auction won & we are attempting to fractionalize the token
- `transferFrom` inside `recover`

Reply:
Solidity will throw an exception if `_nftContract` is not a contract - I believe this can only happen if the `_nftContract` has been self-destructed in between `initialize` and `finalize` or `recover` (because we check that the returns something valid for `ownerOf` during `initialize`)

In the case that the `_nftContract` has been self-destructed, 
- we will never hit  `_fractionalizeNFT` because the auction will be recorded as `LOST`
- `recover` is a moot point because there is no existing NFT to recover

so, I believe we are safe in these two cases

Reply:
i'm fine w/ this changing to be consistent, but to be clear: this line didn't necessarily need to change, correct?

Reply:
the main reason I changed it was so that we ensure that whatever is returned from `ownerOf` function at the time of `initialize` is able to be `decode`'d - tho i think that the Solidity does this under the hood anyway when you use the syntax of calling a contract